.github/workflows/oracle-diff.yml

```
@@ -0,0 +1,289 @@
+      - .github/actions/install-tkrzw/**
+
+permissions:
+  contents: read
+
+env:
+  CARGO_TERM_COLOR: always
+  RUSTFLAGS: -D warnings
+  # The build-oracle.sh work dir, restored or built under a fixed path so the
+  # cached prefix stays valid (libtool/.pc files and the ibus-libpinyin
+  # frontend embed absolute paths at build time).
+  PINYIN_ORACLE_PREFIX: ${{ runner.temp }}/oracle/prefix
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "secrets", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)